forms-flow-api/migrations/versions/cbdc9ba48299_form_logs.py

```diff
@@ -0,0 +1,39 @@
+"""form-logs
+
+Revision ID: cbdc9ba48299
+Revises: e664a2f70952
+Create Date: 2022-11-19 21:33:34.585606
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'cbdc9ba48299'
+down_revision = 'e664a2f70952'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
Based on our last discussion, I thought the form process mapper will only have the active form mapper. And all historic logs regardless of form change or workflow change or title change, it will go into form_logs. For that to happen, we will need more changes than what is here. Not sure if I am on correct page with this PR.
cc: @abhilash-aot @sreehari-aot what do you guys think?

Reply:
Only logs for mapper updates are handled here @sumesh-aot , also the inactive mappers can be moved to historic logs. @shuhaib-aot